com.github.kalibari.audok.json

```
@@ -0,0 +1,93 @@
+{
+    "id" : "com.github.kalibari.audok",
+    "runtime" : "org.gnome.Platform",
+    "sdk": "org.gnome.Sdk",
+    "runtime-version" : "40",
+    "command" : "audok",
+    "rename-desktop-file" : "audok.desktop",
+    "rename-icon" : "audok",
+    "finish-args" : [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=xdg-music",
+        "--filesystem=xdg-download",
+        "--filesystem=xdg-run/pipewire-0:ro",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--socket=wayland"
+    ],
+    "modules" : [
+        {
+            "name": "ffmpeg",
+            "config-opts": [
+              "--enable-rpath",
+              "--enable-gpl",
+              "--disable-static",
+              "--enable-shared",
+              "--disable-doc",
+              "--disable-alsa",
+              "--disable-libx264",
+              "--disable-libx265",
+              "--enable-libmp3lame"
+            ],
+            "sources": [{
+              "type": "archive",
+              "url": "https://www.ffmpeg.org/releases/ffmpeg-4.3.2.tar.xz",
+              "sha256": "46e4e64f1dd0233cbc0934b9f1c0da676008cad34725113fb7f802cfa84ccddb"
+            }],
+            "cleanup": ["/share/ffmpeg/examples"]
+        },
+        {
+            "name": "streamripper",
+            "no-autogen": true,
+            "no-make-install": false,
+            "build-options": {
+                "config-opts": [ "--disable-static" ]
+            },
+            "sources": [
+                {
+                "type": "archive",
+                "url": "https://downloads.sourceforge.net/sourceforge/streamripper/streamripper-1.64.6.tar.gz",
+                "sha512": "8477086d0099f6de861f1c47112476c427073b6e8127bbaaa2dd3e7930fe9a5c6d9b2c68d4a317f1bc0b20c625e665d5c245189049d35468aa83cf51828d4428"
+                },
+                {
+                    "type": "shell",
+                    "commands": [ "cp -p /usr/share/automake-*/config.{sub,guess} ." ]
```

```suggestion
                    "commands": [ 
                        "cp -p /usr/share/automake-*/config.{sub,guess} .",
                        "cp -p /usr/share/automake-*/config.{sub,guess} libmad-0.15.1b/"
                    ]
```

This should fix the build. There is a subproject in the tarball that needed the same treatment.